content/en/docs/collector/internal-telemetry.md

```diff
@@ -179,6 +179,69 @@ Note that the `tracer_provider` section there corresponds to `traces` here.
 [kitchen-sink-config]:
   https://github.com/open-telemetry/opentelemetry-configuration/blob/main/examples/kitchen-sink.yaml
 
+### Self-monitoring
+
+The Collector can be configured to push its own telemetry to an
+[OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)
```

Comment:
Hmm, could we link to docs page, maybe [docs/collector/configuration/#receivers](https://opentelemetry.io/docs/collector/configuration/#receivers). That page links out to the repo's receiver folder.

@theletterf @tiffany76 @open-telemetry/docs-approvers WDYT?

Reply:
Sounds good to me, but I think @jade-guiton-dd's comment below is worth addressing first. This PR's content was purposely removed from the page, and there's been no call to add it back.